src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -309,10 +321,29 @@ impl AstDisplay for Connector {
                     f.write_str("'");
                 }
             }
+            Connector::Postgres {
+                conn,
+                publication,
+                namespace,
+                table,
+                columns,
+            } => {
+                f.write_str("POSTGRES HOST '");
+                f.write_str(&display::escape_single_quote_string(conn));
+                f.write_str("' PUBLICATION '");
+                f.write_str(&display::escape_single_quote_string(publication));
+                f.write_str("' NAMESPACE '");
+                f.write_str(&display::escape_single_quote_string(namespace));
+                f.write_str("' TABLE '");
+                f.write_str(&display::escape_single_quote_string(table));
```

Comment:
There's not an obvious answer to me, but has there been discussion about if this should be an identifier instead of a string? I think the logic is: we *could* do ident because it's postgres and we share ident semantics, but if we want to generalize to other SQL databases that might have different ident semantics, it should be a string because that generalizes to others better?

Reply:
There hasn't been and I see your logic. My rationale was more along the lines "this is not an object of this system, so it must be quoted". But I'm happy to go either way if you think it's more correct to parse it at materialize's side

Reply:
Nah, string is best I think. This is great!